tests/test_vcompressor.py

```diff
@@ -156,3 +156,4 @@ def test_benchmark_basic(self):
                 vcompress_result = self._benchmark_vcompressor(path)
                 self._print_result(filename, original_size,
                                    vcompress_result, other_results, subtest_idx=subtest_idx)
+        self.cleanup()
```

Comment:
The `cmdline_test.py` script is created from line 133, and it is never used. I believe if you just pass `script=None` to `self.template()`, the file won't be created to begin with.

Reply:
OK. I'll correct it.